src/browser/documentation/help/show-users.jsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react'
+import ManualLink from 'browser-components/ManualLink'
+import AdminOnSystemDb from './partials/admin-on-systemdb'
+const title = 'SHOW USERS'
+const subtitle = 'List all available users'
+const category = 'administration'
+const content = (
+  <>
+    <p>
+      Available users can be seen using <code>SHOW USERS</code> which will
+      produce a table of users with four columns: user, roles,
+      passwordChangeRequired, suspended.
+    </p>
+    <div className="links">
+      <div className="link">
+        <p className="title">Reference</p>
+        <p className="content">
+          <ManualLink
+            chapter="cypher-manual"
+            page="/administration/security/users-and-roles/#administration-security-users-show"
+            minVersion="4.0.0"
+          >
+            SHOW USER
```

Comment:
`SHOW USERS`?

Reply:
also I wonder if you can just refer to `title` here for all the files? or is that not always going to be case and/or relevant since  this isn't actually a title?

Reply:
Good catch, thank you! 

The links sometimes point to a different page of the manual that is not necessarily the one for that command (could be a bigger section containing the docs for the command for example).

Reply:
👍 Yeah was just a thought. I guess it doesn't get used that much anyway so not a problem here. If it were something like Redux actions or something else that gets used a lot more it makes more sense as it reduces chance of typos etc